bears/go/GoVetBear.py

```diff
@@ -25,5 +28,96 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         assembly_go_mismatch: bool=False,
+                         useless_assignment: bool=False,
+                         atomic: bool=False,
+                         boolean: bool=False,
+                         bad_build_tag: bool=False,
+                         invalid_cgo: bool=False,
+                         unkeyed_composite_literal: bool=False,
+                         copying_lock: bool=False,
+                         method_signature: bool=False,
+                         nil_function_compare: bool=False,
+                         printf_call: bool=False,
+                         range_loop_variable: bool=False,
+                         shadow_variable: bool=False,
+                         shift: bool=False,
+                         struct_tag_format: bool=False,
+                         unreachable_code: bool=False,
+                         unsafe_pointer: bool=False,
+                         unused_result: bool=False):
+        """
+        :param assembly_go_mismatch:
+            Mismatches between assembly files and Go function declarations.
```

Comment:
Allowed or disallowed when this setting is `True`?

Reply:
This is a type of check. When it is set to True, this check will be performed and the mismatches will not be allowed.

Reply:
Do I need to prefix *disallow* to every param name? :|